config/default.json

```diff
@@ -27,14 +27,17 @@
   },
   "storeTrigger": {
     "url": "http://127.0.0.1:8081",
-    "subUrl": "ingestion"
+    "subUrl": {
```

Comment:
Remove the suburls. We will use it hard-coded since we have more than one suburl. There is no added value to do this configurabilly

Reply:
I think it should be in the configuration and not hard codded because it is the best practice.

Reply:
It is not. you have many subUrls so you had to refactor the code according to those subUrls. There is no added value in making it configurable.